scripts/globals/mobskills/throat_stab.lua

```diff
@@ -19,7 +19,7 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
-    if mob:getFamily() == 271 then -- Jailer of Love, uses only animation.
+    if mob:getPool() == 271 then -- Jailer of Love, uses only animation.
         skill:setMsg(xi.msg.NONE)
         return 0
     end
```

Comment:
This entire section still needs to be deleted

Reply:
this doesn't show up for me anymore locally...